pulpcore/cli/common/generic.py

```diff
@@ -271,6 +271,42 @@ def pulp_option(*args: Any, **kwargs: Any) -> Callable[[_F], _F]:
     type=str,
 )
 
+content_in_option = pulp_option(
+    "--content",
+    "content__in",
+    help=_("Search for {entities} with these content hrefs in them (JSON list)"),
+    callback=load_json_callback,
+)
+
+pulp_created_exact_option = pulp_option(
+    "--created-exact",
```

Comment:
What about "created at"? And do you think we could add some fuzzyness? If i specify the date to the day, i might want to see publications made that day.

On a related note, the others could be named "creted before" and "created after".